kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -1199,7 +1200,7 @@ private void convertRule(RuleOrClaim rule, int ruleIndex, boolean heatCoolEq, St
             sb.append(",\n    ");
             convert(right, sb);
             sb.append(")\n  ");
-            convert(consideredAttributes, rule.att(), sb, freeVarsMap, rule);
+            convert(consideredAttributes, rule.att().add(Att.PRIORITY(), Integer.toString(getPriority(rule.att()))), sb, freeVarsMap, rule);
```

Comment:
This means  that the backends no longer need to calculate priority themselves anymore, right?

Reply:
This is just for macros. The other code for non macros is still in place for now, primarily because the llvm backend computes priorities slightly different than the haskell backend and the k frontend currently.